code/modules/hydroponics/seed_datums.dm

```diff
@@ -184,7 +184,7 @@
 	display_name = "killer tomato plant"
 	mutants = null
 	can_self_harvest = 1
-	has_mob_product = /mob/living/simple_animal/tomato
+	has_mob_product = /mob/living/simple_animal/hostile/tomato
```

Comment:
Looks like in other places you've been removing the hostile path, but here you're adding it. Why is that?